test/integration/http/http_test.go

```diff
@@ -60,4 +71,6 @@ func TestHttpJob(t *testing.T) {
 
 	// Success is the percentage of non-error responses.
 	assert.Assert(t, metrics.Success > 0.95, "too many errors! see the log for details.")
+
+	fmt.Printf("Success!!\n")
```

Comment:
One thing we might need to evaluate is adding or creating some mechanism
to handle logging levels, and stop using `Printf`. If you agree, let's open a separate
issue to take care of that later.

Reply:
yup! agree!
just one comment, the fact that this is a test already gives you some logging control, you only see the printf if you use "-v" for go test.